starlette/websockets.py

```diff
@@ -34,13 +34,20 @@ async def receive(self) -> Message:
         if self.client_state == WebSocketState.CONNECTING:
             message = await self._receive()
             message_type = message["type"]
-            assert message_type == "websocket.connect"
+            if message_type != "websocket.connect":
+                raise RuntimeError(
+                    'WebSocket is not connected. Need to call "accept" first.'
+                )
```

Comment:
What do we think about this kind of style for the `message_type` case?...

`f'Expected ASGI message "websocket.connect", but got {message_type!r}'`